temporal-opentracing/src/main/java/io/temporal/opentracing/internal/DefaultOperationNameAndTagsProvider.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.opentracing.internal;
+
+import com.google.common.collect.ImmutableMap;
+import io.temporal.opentracing.OperationNameAndTagsProvider;
+import io.temporal.opentracing.StandardTagNames;
+import io.temporal.opentracing.StartSpanContext;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DefaultOperationNameAndTagsProvider implements OperationNameAndTagsProvider {
```

Comment:
Let's rename this class. `Default` doesn't provide any information about this strategy. 
You can put a note that it's the default strategy in the javadoc.
It can be something like `OperationTypeWithActionNameAsOperationName` instead.
Where OperationType means RunActivity, etc, and ActionName means workflow or activity type.
Also, there should be a javadoc explaining the output of this strategy with a couple of examples.